src/solvers/dgsem_tree/dg_2d.jl

```diff
@@ -1036,6 +1097,148 @@ function calc_mortar_flux!(surface_flux_values,
     return nothing
 end
 
+function calc_flux_correction!(surface_flux_values,
```

Comment:
Maybe add a comment noting that this is called after the standard mortar flux is computed?